doc/src/fix_pimd_bosonic.rst

```diff
@@ -0,0 +1,232 @@
+:ref:`(Higer) <HigerFeldman>`.
+
+Restart, fix_modify, output, run start/stop, minimize info
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+The use of :doc:`binary restart files <restart>` and :doc:`fix_modify
+<fix_modify>` is the same as in :doc:`fix pimd <fix_pimd>`.
+
+Fix *pimd/nvt/bosonic* computes a global 4-vector, which can be accessed by
+various :doc:`output commands <Howto_output>`.  The quantities in
+the global vector are:
+
+   #. the total spring energy of the quasi-beads,
+   #. the current temperature of the classical system of ring polymers,
+   #. the current value of the scalar virial estimator for the kinetic
+      energy of the quantum system :ref:`(Herman) <HermanBB>` (see the justification in the supporting information of :ref:`(Hirshberg2) <HirshbergInvernizzi>`),
+   #. the current value of the scalar primitive estimator for the kinetic
+      energy of the quantum system :ref:`(Hirshberg1) <Hirshberg>`.
+
+The vector values calculated by fix *pimd/nvt/bosonic* are "extensive", except
+for the temperature, which is "intensive".
+
+Fix *pimd/langevin/bosonic* computes a global 6-vector, which can be accessed
+by various :doc:`output commands <Howto_output>`. The quantities in the
+global vector are:
+
+   #. kinetic energy of the beads,
+   #. spring elastic energy of the beads,
+   #. potential energy of the bead,
+   #. total energy of all beads (conserved if *ensemble* is *nve*) if *esynch* is *yes*
+   #. primitive kinetic energy estimator :ref:`(Hirshberg1) <Hirshberg>`
+   #. virial energy estimator :ref:`(Herman) <HermanBB>` (see the justification in the supporting information of :ref:`(Hirshberg2) <HirshbergInvernizzi>`).
+
+The first three are different for different log files, and the others
+are the same for different log files, except for the primitive kinetic
+energy estimator when setting *esynch* to *no*. Then, the primitive
+kinetic energy estimator is obtained by summing over all log files.
+Also note that when *esynch* is set to *no*, the fourth output gives the
+total energy of all beads excluding the spring elastic energy; the total
+classical energy can then be obtained by adding the sum of second output
+over all log files.  All vector values calculated by fix
+*pimd/langevin/bosonic* are "extensive".
+
+For both *pimd/nvt/bosonic* and *pimd/langevin/bosonic*, the contribution of the
+exterior spring to the primitive estimator is printed to the first log
+file.  The contribution of the :math:`P-1` interior springs is printed
+to the other :math:`P-1` log files.  The contribution of the constant
+:math:`\frac{PdN}{2 \beta}` (with :math:`d` being the dimensionality) is
+equally divided over log files.
+
+Restrictions
+""""""""""""
+
+These fixes are part of the REPLICA package.  They are only enabled if
+LAMMPS was built with that package.  See the :doc:`Build package
+<Build_package>` page for more info.
+
+The restrictions of :doc:`fix pimd <fix_pimd>` apply.
+
+Default
```

Comment:
Typically, all command doc pages have a "Related commands" section after Restrictions and before Default.
The one here should provide links to the 2 older PIMD commands.  And you should add links to your 2 new commands to the Related section on the old doc page.